<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>BabyMEGSQUIDControlDgl</class>
 <widget class="QDialog" name="BabyMEGSQUIDControlDgl">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>900</width>
    <height>901</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>FLL Controller</string>
  </property>
  <property name="autoFillBackground">
   <bool>true</bool>
  </property>
  <layout class="QGridLayout" name="gridLayout_4">
   <item row="2" column="0">
    <widget class="QGroupBox" name="groupBox_2">
     <property name="styleSheet">
      <string notr="true">color: rgb(0, 0, 0);
background-color:rgb(216, 216, 216);</string>
     </property>
     <property name="title">
      <string>MessageBox</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_11">
      <item row="0" column="0">
       <layout class="QGridLayout" name="gridLayout_10">
        <item row="0" column="0">
         <widget class="QTextEdit" name="m_tx_info">
          <property name="styleSheet">
           <string notr="true">background-color: rgb(255, 255, 255);</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QTabWidget" name="tabWidget">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="autoFillBackground">
      <bool>false</bool>
     </property>
     <property name="styleSheet">
      <string notr="true"/>
     </property>
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="tab_SQUIDCONTROL">
      <attribute name="title">
       <string>SQUID Control</string>
      </attribute>
      <layout class="QGridLayout" name="gridLayout_8">
       <item row="0" column="1" rowspan="3">
        <widget class="QGroupBox" name="groupBox">
         <property name="styleSheet">
          <string notr="true"/>
         </property>
         <property name="title">
          <string>Parameters</string>
         </property>
         <layout class="QGridLayout" name="gridLayout_6">
          <item row="2" column="0">
           <layout class="QGridLayout" name="gridLayout_3">
            <item row="0" column="3">
             <widget class="QPushButton" name="m_Qbn_mirco_reset">
              <property name="enabled">
               <bool>false</bool>
              </property>
              <property name="text">
               <string>MicroReset</string>
              </property>
             </widget>
            </item>
            <item row="0" column="2">
             <widget class="QPushButton" name="m_Qbn_int_reset">
              <property name="enabled">
               <bool>false</bool>
              </property>
              <property name="text">
               <string>Integrator Reset</string>
              </property>
             </widget>
            </item>
            <item row="0" column="1">
             <widget class="QPushButton" name="m_Qbn_amp">
              <property name="text">
               <string>Amp?</string>
              </property>
             </widget>
            </item>
            <item row="0" column="0">
             <widget class="QPushButton" name="m_Qbn_tunecheck">
              <property name="text">
               <string>TuneCheck</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item row="0" column="0">
           <layout class="QGridLayout" name="gridLayout_2">
            <item row="3" column="1">
             <widget class="QComboBox" name="m_Qcb_slew">
              <item>
               <property name="text">
                <string>Slow</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>Normal</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>Fast</string>
               </property>
              </item>
             </widget>
            </item>
            <item row="2" column="1">
             <widget class="QComboBox" name="m_Qcb_pregain">
              <item>
               <property name="text">
                <string>x1</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>x2</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>x5</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>x10</string>
               </property>
              </item>
             </widget>
            </item>
            <item row="0" column="0">
             <widget class="QLabel" name="label_4">
              <property name="text">
               <string>High Pass</string>
              </property>
             </widget>
            </item>
            <item row="1" column="1">
             <widget class="QComboBox" name="m_Qcb_lp">
              <item>
               <property name="text">
                <string>Off</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>200Hz</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>500Hz</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>1KHz</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>5KHz</string>
               </property>
              </item>
             </widget>
            </item>
            <item row="1" column="0">
             <widget class="QLabel" name="label_5">
              <property name="text">
               <string>Low Pass</string>
              </property>
             </widget>
            </item>
            <item row="0" column="1">
             <widget class="QComboBox" name="m_Qcb_hp">
              <item>
               <property name="text">
                <string>Off</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>0.16Hz</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>1Hz</string>
               </property>
              </item>
             </widget>
            </item>
            <item row="3" column="0">
             <widget class="QLabel" name="label_7">
              <property name="text">
               <string>Slew</string>
              </property>
             </widget>
            </item>
            <item row="4" column="0">
             <widget class="QLabel" name="label_14">
              <property name="text">
               <string>Heat Time (msec)</string>
              </property>
             </widget>
            </item>
            <item row="1" column="4">
             <widget class="QComboBox" name="m_Qcb_reset_lock">
              <item>
               <property name="text">
                <string>Off</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>On</string>
               </property>
              </item>
             </widget>
            </item>
            <item row="0" column="4">
             <widget class="QComboBox" name="m_Qcb_auto_reset">
              <item>
               <property name="text">
                <string>Off</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>On</string>
               </property>
              </item>
             </widget>
            </item>
            <item row="1" column="3">
             <widget class="QLabel" name="label_9">
              <property name="text">
               <string>Reset Lock</string>
              </property>
             </widget>
            </item>
            <item row="0" column="3">
             <widget class="QLabel" name="label_8">
              <property name="text">
               <string>Auto Reset</string>
              </property>
             </widget>
            </item>
            <item row="4" column="1">
             <widget class="QSpinBox" name="m_Qsb_heattime">
              <property name="maximum">
               <number>10000</number>
              </property>
             </widget>
            </item>
            <item row="2" column="0">
             <widget class="QLabel" name="label_6">
              <property name="text">
               <string>PreGain</string>
              </property>
             </widget>
            </item>
            <item row="5" column="0">
             <widget class="QLabel" name="label_15">
              <property name="text">
               <string>Cool Time(msec)</string>
              </property>
             </widget>
            </item>
            <item row="5" column="1">
             <widget class="QSpinBox" name="m_Qsb_cooltime">
              <property name="maximum">
               <number>1000000</number>
              </property>
             </widget>
            </item>
            <item row="3" column="3">
             <widget class="QLabel" name="label_11">
              <property name="text">
               <string>Offset </string>
              </property>
             </widget>
            </item>
            <item row="4" column="3">
             <widget class="QLabel" name="label_12">
              <property name="text">
               <string>Bias</string>
              </property>
             </widget>
            </item>
            <item row="5" column="3">
             <widget class="QLabel" name="label_13">
              <property name="text">
               <string>Modulation</string>
              </property>
             </widget>
            </item>
            <item row="2" column="3">
             <widget class="QLabel" name="label_18">
              <property name="text">
               <string>PostGain</string>
              </property>
             </widget>
            </item>
            <item row="2" column="4">
             <widget class="QComboBox" name="m_Qcb_postgain">
              <item>
               <property name="text">
                <string>x1</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>x10</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>x100</string>
               </property>
              </item>
             </widget>
            </item>
            <item row="0" column="5">
             <widget class="QLabel" name="label_10">
              <property name="text">
               <string/>
              </property>
             </widget>
            </item>
            <item row="3" column="4">
             <widget class="QSpinBox" name="m_Qsb_offset">
              <property name="maximum">
               <number>9999</number>
              </property>
             </widget>
            </item>
            <item row="4" column="4">
             <widget class="QSpinBox" name="m_Qsb_bias">
              <property name="maximum">
               <number>9999</number>
              </property>
             </widget>
            </item>
            <item row="5" column="4">
             <widget class="QSpinBox" name="m_Qsb_mod">
              <property name="maximum">
               <number>9999</number>
              </property>
             </widget>
            </item>
            <item row="4" column="2">
             <widget class="QPushButton" name="m_Qbn_heattime">
              <property name="text">
               <string>Set </string>
              </property>
             </widget>
            </item>
            <item row="5" column="2">
             <widget class="QPushButton" name="m_Qbn_cooltime">
              <property name="text">
               <string>Set</string>
              </property>
             </widget>
            </item>
            <item row="3" column="5">
             <widget class="QPushButton" name="m_Qbn_offset">
              <property name="text">
               <string>Set</string>
              </property>
             </widget>
            </item>
            <item row="4" column="5">
             <widget class="QPushButton" name="m_Qbn_bias">
              <property name="text">
               <string>Set</string>
              </property>
             </widget>
            </item>
            <item row="5" column="5">
             <widget class="QPushButton" name="m_Qbn_mod">
              <property name="text">
               <string>Set</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item row="1" column="0">
           <widget class="QLabel" name="label_19">
            <property name="text">
             <string/>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item row="3" column="0">
        <widget class="QLabel" name="label_16">
         <property name="text">
          <string>Tune Graph</string>
         </property>
        </widget>
       </item>
       <item row="0" column="0">
        <layout class="QGridLayout" name="gridLayout_7">
         <item row="1" column="0" colspan="2">
          <widget class="QPushButton" name="m_Qbn_SyncGUI">
           <property name="text">
            <string>Sync GUI</string>
           </property>
          </widget>
         </item>
         <item row="6" column="0">
          <widget class="QLabel" name="label_2">
           <property name="text">
            <string>Channel</string>
           </property>
          </widget>
         </item>
         <item row="6" column="1">
          <widget class="QComboBox" name="m_Qcb_channel"/>
         </item>
         <item row="5" column="1">
          <widget class="QComboBox" name="m_Qcb_commtype">
           <item>
            <property name="text">
             <string>Broadcast</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>GroupA</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>GroupB</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>GroupC</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>GroupD</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>GroupE</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>GroupF</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>GroupG</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>GroupH</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>Single Channel</string>
            </property>
           </item>
          </widget>
         </item>
         <item row="5" column="0">
          <widget class="QLabel" name="label">
           <property name="text">
            <string>CommType</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item row="1" column="0" rowspan="2">
        <layout class="QGridLayout" name="gridLayout">
         <item row="0" column="0">
          <widget class="QLabel" name="label_3">
           <property name="text">
            <string>Operate Mode</string>
           </property>
          </widget>
         </item>
         <item row="0" column="1">
          <widget class="QComboBox" name="m_Qcb_opermode">
           <item>
            <property name="text">
             <string>Choose</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>Run</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>Tune</string>
            </property>
           </item>
          </widget>
         </item>
         <item row="1" column="0">
          <widget class="QPushButton" name="m_Qbn_retune">
           <property name="text">
            <string>Retune</string>
           </property>
          </widget>
         </item>
         <item row="2" column="0">
          <widget class="QPushButton" name="m_Qbn_heat">
           <property name="text">
            <string>Heat</string>
           </property>
          </widget>
         </item>
         <item row="3" column="0">
          <widget class="QPushButton" name="m_Qbn_atune">
           <property name="text">
            <string>ATune</string>
           </property>
          </widget>
         </item>
         <item row="4" column="0">
          <widget class="QPushButton" name="m_Qbn_reset">
           <property name="text">
            <string>Reset</string>
           </property>
          </widget>
         </item>
         <item row="5" column="0">
          <widget class="QPushButton" name="m_Qbn_heatTune">
           <property name="text">
            <string>Heat &amp;&amp; Tune</string>
           </property>
          </widget>
         </item>
         <item row="1" column="1">
          <widget class="QPushButton" name="m_Qbn_save">
           <property name="text">
            <string>Save</string>
           </property>
          </widget>
         </item>
         <item row="2" column="1">
          <widget class="QPushButton" name="m_Qbn_save1">
           <property name="text">
            <string>%$Save</string>
           </property>
          </widget>
         </item>
         <item row="3" column="1">
          <widget class="QPushButton" name="m_Qbn_groupheat">
           <property name="text">
            <string>Group Heat</string>
           </property>
          </widget>
         </item>
         <item row="4" column="1">
          <widget class="QPushButton" name="m_Qbn_last">
           <property name="text">
            <string>Last</string>
           </property>
          </widget>
         </item>
         <item row="5" column="1">
          <widget class="QPushButton" name="m_Qbn_default">
           <property name="text">
            <string>Default</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item row="5" column="0" colspan="2">
        <layout class="QGridLayout" name="gridLayout_13">
         <item row="0" column="0">
          <widget class="QComboBox" name="m_Qcb_bar_graph_select">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <item>
            <property name="text">
             <string>Triangle Amplitudes</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>AutoReset</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>Biases</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>High Passes</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>Low Passes</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>Modes</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>Modulations</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>Offsets</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>Reset</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>Slews</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>PreGains</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>PostGains</string>
            </property>
           </item>
          </widget>
         </item>
         <item row="1" column="0">
          <widget class="QTableWidget" name="tbw_parameters"/>
         </item>
        </layout>
       </item>
       <item row="4" column="0" colspan="2">
        <layout class="QFormLayout" name="lay_tune">
         <property name="sizeConstraint">
          <enum>QLayout::SetFixedSize</enum>
         </property>
         <property name="fieldGrowthPolicy">
          <enum>QFormLayout::ExpandingFieldsGrow</enum>
         </property>
         <property name="horizontalSpacing">
          <number>0</number>
         </property>
        </layout>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tab_SQUID_STAT">
      <attribute name="title">
       <string>SQUID Stat</string>
      </attribute>
      <layout class="QGridLayout" name="gridLayout_9">
       <item row="0" column="0">
        <layout class="QGridLayout" name="gridLayout_5">
         <item row="4" column="0">
          <widget class="QLabel" name="label_17">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="text">
            <string>SQUID STAT</string>
           </property>
          </widget>
         </item>
         <item row="5" column="0">
          <widget class="QTextBrowser" name="textBrowser">
           <property name="styleSheet">
            <string notr="true">background-color: rgb(220, 220, 220);</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
   <item row="3" column="0">
    <layout class="QFormLayout" name="formLayout_4">
     <property name="fieldGrowthPolicy">
      <enum>QFormLayout::AllNonFixedFieldsGrow</enum>
     </property>
     <property name="horizontalSpacing">
      <number>6</number>
     </property>
     <property name="verticalSpacing">
      <number>1</number>
     </property>
     <item row="0" column="0">
      <widget class="QPushButton" name="m_Qbn_Cancel">
       <property name="text">
        <string>Close</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
